x/genutil/client/cli/export.go

```diff
@@ -70,6 +70,7 @@ func ExportCmd(appExporter servertypes.AppExporter) *cobra.Command {
 				return err
 			}
 			defer cleanup()
+			defer traceWriter.Close()
```

Comment:
I think you need to nil the check the traceWriter: https://github.com/cosmos/cosmos-sdk/actions/runs/6713065660/job/18243935809?pr=18322